contributor-docs/adrs/adr-004-children-as-api.md

```diff
@@ -0,0 +1,403 @@
+# ADR 004: Strict props or Composite components
+
+## Status
+
+Proposed
+
+<br/>
+
+_Note: These are in no way rules, these should be treated as a starting point for API conversations._
+
+_Note #2: Consumer is used multiple times on this page. It refers to the developers consuming the component API and not end users._
+
+<br/>
+
+## Prefer using children for “content”
+
+With React, `children` is the out-of-the-box way for putting _content_ inside your component. By using `children` instead of our own custom prop, we can make the API “predictable” for its consumers.
```

Comment:
> ...for putting _content_ inside...

Do we want to be more specific about what we mean by "content"?

> ...for putting [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content) inside...

Reply:
ooh TIL phrasing content!